packages/server/lib/controllers/xhrs.coffee

```diff
@@ -50,6 +50,11 @@ module.exports = {
         if _.isObject(data)
           data = JSON.stringify(data)
 
+        ## when data is null, JSON.stringify returns null.
+        ## handle that case. 
+        if data is null
+          data = []
```

Comment:
why `[]` and not `''`?

Reply:
It's because I didn't read the documentation carefully. It starts with the function with array argument and [later string argument](https://nodejs.org/api/buffer.html#buffer_class_method_buffer_from_string_encoding). 

Changed it to `''`.